cni/network/network.go

```diff
@@ -235,11 +290,17 @@ func (plugin *netPlugin) Add(args *cniSkel.CmdArgs) error {
 	}
 
 	// Initialize endpoint info.
-	epInfo := &network.EndpointInfo{
+	epInfo = &network.EndpointInfo{
 		Id:          endpointId,
 		ContainerID: args.ContainerID,
 		NetNsPath:   args.Netns,
 		IfName:      args.IfName,
+		// Windows only DNS info.
+		DNS: network.DNSInfo{
+			Suffix:  k8sNamespace + "." + strings.Join(nwCfg.DNS.Search, ","),
```

Comment:
something have to be changed here if nwcfg.dns.search is empty then suffix will get wrong

Reply:
done.